conductor/pom.xml

```diff
@@ -9,12 +9,59 @@
         <version>0.5-SNAPSHOT</version>
     </parent>
 
+    <properties>
+      <spark.version>1.4.1</spark.version>
+      <hadoop.version>2.2.0</hadoop.version>
+    </properties>
```

Comment:
I think we may either want to use Spark 1.2.1 and whatever Hadoop version Spark 1.2.1 was built against (making this PR a no-op) OR 1.5.2/2.6.0.